is_iot_collector/main.py

```diff
@@ -1,56 +1,65 @@
+import os
 import time
 import utils
+import yaml
+import logging
+import logging.config
 from is_iot_collector.readings.soil_moisture import soil_moisture
 from is_iot_collector.readings.air_properties import air_properties
 from is_iot_collector.readings.light_intensity import light_intensity
 from is_iot_collector.mqtt.mqtt_publisher import mqtt_client
 from is_iot_collector.local_tinydb import tiny_db
 from is_iot_collector.json_builder import *
-from logger import LOG
+
+
+def init_logger(file_path: str):
+    with open(file_path, 'r') as f:
+        config = yaml.safe_load(f.read())
+    config['handlers']['file_handler']['filename'] = f"{os.getenv('PROJECT_PATH')}/{config['handlers']['file_handler']['filename']}"
+
+    logging.config.dictConfig(config)
+
 
 def main():
+    init_logger(os.getenv('PROJECT_PATH') + '/logger_config.yml')
     reading_time = int(utils.get_setting('readingTime'))
     register_time = int(utils.get_setting('registerTime'))
     register_expires_at = time.time() + register_time
 
-    output_file = utils.find_next_output_file()
-
-    while(True):
+    while True:
         if time.time() > register_expires_at:
             mqtt_client.register()
             register_expires_at = time.time() + register_time
 
         jdata = JsonBuilder()
         moisture = soil_moisture.percent_all_values()
-        if moisture != None:
+        if moisture is not None:
```

Comment:
can't use here if moisture?
easyer to read imo

Reply:
definitely can be used only `if moisture:` this is how the IDE refactored the expression since it's using None